meta-refkit-industrial/recipes-ros/cv-bridge/cv-bridge_%.bbappend

```diff
@@ -0,0 +1,3 @@
+# TODO: This way cv_bridge doesn't require libboost_python, but this tweak
+#       should be dropped after ROS is compatible with python3-only distro.
+EXTRA_OECMAKE_append_refkit-industrial = " -DANDROID=ON"
```

Comment:
#109 is unrelated to this. Without this option cv_bridge wants to build python2 bindings which depend on python2 bindings to Boost (disabled in oe-core). The proper solution would be to rewrite cv_bridge's code to support python3.